db/models/Reviews.js

```diff
@@ -0,0 +1,41 @@
+const db = require("../db");
+const { Sequelize } = db;
+
+const Review = db.define("review", {
+    id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+    },
+    content: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+        validate: {
+            notEmpty: {
+                args: true,
+                msg: "please provide a review comment"
+            }
+        }
+    },
+    stars: {
+        type: Sequelize.ENUM,
```

Comment:
This works but it seems redundant to define an enum, then enforce it using a min-max.
You would achieve the same thing by defining this as type: Sequelize.INTEGER, and just keep your min, max and isInt validations.